### PR TITLE
More terse naming for WGSL items/segments.

### LINKED ISSUE #59: Simplify types, and make private and public apis more consistant

### PR DESCRIPTION
**Pull request by iwoplaza**
Resolved #59

### CODE REVIEW

File: packages/wigsill/src/std140/types.ts [hunk 1]
```diff
@@ -23,12 +23,12 @@ export type WGSLValue<TDataType> = {
   readonly __dataType: TDataType;
 };
 
-export type AnyWGSLPointerType = WGSLPointerType<'function', AnyWGSLDataType>;
+export type AnyWGSLPointerType = WGSLPointerType<'function', AnyWgslData>;
```
Comment (mhawryluk) [inline packages/wigsill/src/std140/types.ts]: -> AnyWgslPointer & -> WgslPointer ?

File: packages/wigsill/src/std140/types.ts [hunk 2]
```diff
@@ -23,12 +23,12 @@ export type WGSLValue<TDataType> = {
...
 
-export type WGSLFnArgument = AnyWGSLPointerType | AnyWGSLDataType;
+export type WGSLFnArgument = AnyWGSLPointerType | AnyWgslData;
```
Comment (mhawryluk) [inline packages/wigsill/src/std140/types.ts]: -> WgslFnArgument

File: packages/wigsill/src/wgslBuffer.ts [hunk 3]
```diff
@@ -1,25 +1,46 @@
...
+// ----------
+// Public API
+// ----------
+
+export interface WgslBuffer<TData extends AnyWgslData>
+  extends WgslResolvable,
+    WgslAllocatable<TData> {
+  alias(label: string): WgslBuffer<TData>;
+
+  write(runtime: WGSLRuntime, data: Parsed<TData>): void;
```
Comment (mhawryluk) [inline packages/wigsill/src/wgslBuffer.ts]: WgslRuntime

File: packages/wigsill/src/wgslFunction.ts [hunk 4]
```diff
@@ -1,30 +1,30 @@
...
-import type { ResolutionCtx, WGSLItem, WGSLSegment } from './types';
-import { type WGSLCode, code } from './wgslCode';
-import { WGSLIdentifier, identifier } from './wgslIdentifier';
+import type { ResolutionCtx, Wgsl, WgslResolvable } from './types';
+import { code } from './wgslCode';
+import { WgslIdentifier } from './wgslIdentifier';
 
 type ValuesFromTypes<TArgTypes extends WGSLFnArgument[]> = {
-  [K in keyof TArgTypes]: WGSLValue<TArgTypes[K]> & WGSLIdentifier;
+  [K in keyof TArgTypes]: WGSLValue<TArgTypes[K]> & WgslIdentifier;
```
Comment (mhawryluk) [inline packages/wigsill/src/wgslFunction.ts]: WgslValue, WgslFunctionCall, WgslFunction

File: packages/wigsill/src/wgslBuffer.ts (hunk 3 above)
Comment (iwoplaza) [reply packages/wigsill/src/wgslBuffer.ts]: Maybe since `Wgsl` is usually reserved for building blocks of a program, it should be named `WigsillRuntime`?

**Pull request merged**